bundles/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/InfluxDBPersistenceService.java

```diff
@@ -101,8 +124,9 @@ public InfluxDBPersistenceService(final @Reference ItemRegistry itemRegistry,
         if (configuration.isValid()) {
             this.influxDBRepository = createInfluxDBRepository();
             this.influxDBRepository.connect();
-            this.itemToStorePointCreator = new ItemToStorePointCreator(configuration, influxDBMetadataService);
-            tryReconnection = true;
+            this.storeJob = ThreadPoolManager.getScheduledPool("org.openhab.influxdb")
+                    .scheduleWithFixedDelay(this::doStore, 1, 1, TimeUnit.SECONDS);
```

Comment:
I am asking myself if 1 second is the good frequency ?
We have to find the good compromise between the size of the queue and how much often the data is stored in DB.

Reply:
What do you suggest? The longer the time, the greater the risk of losing data. IMO 10s is far too long, but as you suggest, maybe 1s is too small.

Reply:
Is there really a risk to loose data ?? Why ?

I agree that 10s is too high.

Reply:
Power loss e.g.. I agree that it's not very likely, but the risk is there. What do you think, shall we try 3s?

Reply:
Yes, I was thinking about 3s.